routes/web.php

```diff
@@ -15,7 +15,9 @@
 |
 */
 
-
+Route::get('/debug-sentry', function () {
```

Comment:
Remove this temporary route.

Reply:
merge to develop branch? fix the comment means previous once?